packages/zoltan2/test/connectivity/meshData/noedges-answers.txt

```diff
@@ -0,0 +1,4 @@
+1
```

Comment:
This file says all four vertices should be removed.  But the basil friction file says each point is grounded.  Grounded points should not be removed, right?  If the code removes them, that would be an error.

Reply:
From my understanding, standalone vertices in the application are to be removed. Even though each vertex is grounded, there is no other connection to ground so each vertex should be removed.

Reply:
In other words, being initially grounded does not guarantee that the vertex will be kept. Similarly, a mesh element with a single vertex that is initially grounded would have all vertices in that element removed.